### ISSUE TITLE
Deepwalk: Command not found

### ISSUE THREAD

**Issue by YutingSun163** [phanein/deepwalk#96]
I have installed deepwalk as the instruction says, and my python version is 3.7.0. However, when I run deepwalk command, it says "deepwalk: command not found"

**Comment by GTmac** [comment]
Looking at the more recent issue you opened, I assume that you have already fixed this. Should I close this issue?

**Comment by YutingSun163** [comment]
> Looking at the more recent issue you opened, I assume that you have already fixed this. Should I close this issue?
No, I didn't fix it. And I also tried upgrade gensim to the latest version, but it won't work. I tried it on both Python2 and Python3. But I have found a solution, that is, to import the deepwalk package on pycharm, so I can run my code, and it worked well when I use Python2 on pycharm, but still "deepwalk: command not found" on Python3 (

**Comment by GTmac** [comment]
I do not have pycharm on my machine, so I am not sure about what happened. Also, when you report an issue, please provide the exact commands you run, outputs of these commands and your environment otherwise it would be very hard for me to identify what could be the problem.
